.vscode/settings.json

```diff
@@ -34,5 +34,5 @@
   "typescript.format.insertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces": true,
   "editor.formatOnSave": true,
   "typescript.tsdk": "./common/temp/node_modules/typescript/lib",
-  "tslint.autoFixOnSave": true
+  "tslint.autoFixOnSave": false
```

Comment:
I keep seeing mistakes accidentally introduced by this!

For example; I needed to define an `any` to clean up an ambiguous and missing typing. Saving the file converted it into `{}` which wasted time multiple times when I didn't realize it happened.

So I'm not quite sure this one is good in particular. Sometimes it's better to just fail on the lint at build so you know what you're doing wrong.